diktat-rules/src/test/resources/test/smoke/src/main/kotlin/Example8Expected.kt

```diff
@@ -0,0 +1,14 @@
+package org.cqfn.diktat
+
+fun foo() {
+    val sum: (Int, Int, Int,) -> Int = fun(
```

Comment:
please also add a unit test, even duplicated. Not only smoke is needed

Reply:
Added